runtime/zeitgeist/src/parameters.rs

```diff
@@ -288,6 +288,35 @@ parameter_types! {
     /// Pallet identifier, mainly used for named balance reserves. DO NOT CHANGE.
     pub const TreasuryPalletId: PalletId = PalletId(*b"zge/tsry");
 
+    // Bounties
+    /// The amount held on deposit for placing a bounty proposal.
+    pub const BountyDepositBase : Balance = 10 * BASE;
```

Comment:
This seems a bit low to me. On Polkadot it is about 7$ currently, on Kusama 50$. I propose we use `100 * BASE` for about 10$.
```suggestion
    pub const BountyDepositBase : Balance = 100 * BASE;
```